### ISSUE TITLE
Can't fetch docker image from ECR

### ISSUE THREAD

**Issue by Fazik** [hashicorp/packer#7764]
- Packer version from `packer version`
Packer v1.4.1
- Host platform
   Ubuntu 16.04
- Debug log output from `PACKER_LOG=1 packer build template.json`.
  https://gist.github.com/Fazik/800eb3a8510322fb5de2c3d591464c49
- The _simplest example template and scripts_ needed to reproduce the bug.
 https://gist.github.com/Fazik/bb5f0bbe0390a0b75f3b7c5171974eb2
- Build command:
`packer build -var 'aws_access_key=MY_KEY' -var 'aws_secret_key=MY_SECRET' -var 'ecr_url=ACCOUNTID.dkr.ecr.eu-west-1.amazonaws.com' -var 'user_build=andrii' -var 'release=latest' -var 'role=jetty' -on-error=ask stacks.json`

Basically when I'm trying to pull image from ECR - build fails
repository base exists and tag latest in it also present.
Maybe I'm doing something wrong, but maybe there is some bug.

**Comment by rickard-von-essen** [comment]
If you change to:

```json
"image": "{{ user `ecr_url` }}/base:{{user `release`}
```

it will work.

**Comment by Fazik** [comment]
Oh..
Sorry, I'm just too new in packer)
Many thanks, @rickard-von-essen
